client_test.go

```diff
@@ -2161,3 +2161,215 @@ func TestEmptyCapabilities(t *testing.T) {
 		}(tt)
 	}
 }
+
+func TestVipsModsRequest(t *testing.T) {
+	t.Parallel()
+	var receivedMsg string
+	ch := make(chan struct{})
+
+	host := startServerMultiConns(t, 2, nothingOnConnect, func(message string) {
+		if strings.HasPrefix(message, "PRIVMSG") {
+			receivedMsg = message
+			// the fact we close this and it doesnt panic means that only one message is received.
+			close(ch)
+		}
+	})
+
+	client := newTestClient(host)
+
+	go client.Connect()
```

Comment:
why did this fail when i literally copied it from the test above?

Reply:
The linting is on a "lint new things, but not old things" since it was implemented. See other tests, e.g. `connectAndEnsureGoodDisconnect` (which you might want to use anyway)